### PR TITLE
Fixed magnetorquer module & test

### PR DESCRIPTION
**Pull request by ron-popov**
Fixed the magnetorquer's gpio pins not working and wrote a test for it.

### CODE REVIEW

**Comment by ron-popov** [comment]
Review ? @Rogozin-high-school/core-developers

File: Tests/hBridge.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from ..Modules.Magnetorquer.hBridge import hBridge
+
+actuator = hBridge(23, 24, 14, 15)
+
+while True:
+    inp = input('Enter direction : ')
```
Comment (yotam180) [inline Tests/hBridge.py:6]: Can we actually take input on the satellite side?

**Review by yotam180 — APPROVED** [review]
Just a question, tho...

File: Tests/hBridge.py (hunk 1 above)
Comment (ron-popov) [reply Tests/hBridge.py:6]: It's a test, it's not intended for use in the experiments only for testing each module.

**Pull request merged**